benches/wasm/mod.rs

```diff
@@ -57,51 +57,6 @@ pub fn protobuf(c: &mut Criterion) {
     });
 }
 
-pub fn drop(criterion: &mut Criterion) {
```

Comment:
@Hoverbear I'm curious if you have any thoughts here. This benchmark currently hangs forever as `bench_group_transforms_over_parameterized_event_sizes` determines when an event has been processed by polling the output of the transform (which in this case, no event is emitted). I could not determine a good way to otherwise determine when an event has been fully processed by a transform.

Reply:
Ohhhh, I think we need to just drop this for now then. :( With the task transform changes I think the only assertion we can make is that "nothing ever arrives" which seems to be passing right now.